src/main/resources/db/changelog/changes/04-insert-roles.yaml

```
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-roles
+      author: dmytro

```

```suggestion
      author: dmytro_potapov
```
it's better to provide full name as author of changeSet